### PR TITLE
chore: update auction result list item component

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [CX-3226] 

### Description

This PR updates the auction result list item component to match the new designs

https://user-images.githubusercontent.com/11945712/208158210-5ad0e897-d0cb-4cd3-b8e9-d5040948afed.mov

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- update auction result list item component - mounir

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-3226]: https://artsyproduct.atlassian.net/browse/CX-3226?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Components/Lists/AuctionResultListItem.tsx [hunk 1]
```diff
@@ -69,69 +76,69 @@ const AuctionResultListItem: React.FC<Props> = ({
...
                 {auctionResult.title}
-                <Text variant="xs" color="black60">
-                  {!!auctionResult.dateText &&
-                    auctionResult.dateText !== "" &&
-                    `, ${auctionResult.dateText}`}
-                </Text>
+                {!!auctionResult.dateText &&
+                  auctionResult.dateText !== "" &&
+                  `, ${auctionResult.dateText}`}
+                {/* </Text> */}
```
Comment (MrSltun) [inline src/app/Components/Lists/AuctionResultListItem.tsx:94]:
```suggestion
```

File: src/app/Components/Lists/AuctionResultListItem.tests.tsx [hunk 2]
```diff
@@ -76,7 +80,7 @@ describe("AuctionResults", () => {
     expect(tree.findAllByProps({ testID: "priceUSD" }).length).toBe(0)
   })
 
-  it("renders price in USD when currency is not USD", () => {
+  it.only("renders price in USD when currency is not USD", () => {
```
Comment (MrSltun) [inline src/app/Components/Lists/AuctionResultListItem.tests.tsx:83]: 👀

**Review by MrSltun — COMMENTED** [review]
Looks great! Thank you, Mounir! 👏 👏 
I just left a couple of comments, also I believe this work should be behind a feature flag (according to the ticket)

**Review by dariakoko — DISMISSED** [review]
Looks god, thank you Mounir
Let's resolve Sultan's comments and merge

**Pull request merged**